Sources/TensorFlow/Layers/Core.swift

```diff
@@ -105,10 +108,6 @@ public struct Reshape<Scalar: TensorFlowFloatingPoint>: Layer {
     /// The target shape.
     @noDerivative public let shape: Tensor<Int32>
 
-    // TF-331 workaround:
-    @usableFromInline
-    internal var _nontrivial = Tensor<Float>(0)
```

Comment:
I'd suggest keeping this workaround because the system isn't ready for it yet. I'm about to fix it today.

Reply:
I only removed it because all tests pass without it locally, but I'll add it back. :)